app/models/alma_pod_records/alma_pod_job.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+module AlmaPodRecords
+  class AlmaPodJob < LibJob
+    attr_reader :documents
+
+    def initialize(incoming_file_list: nil, file_pattern: '\.tar\.gz$', since: nil)
+      super(category: 'AlmaPodRecords')
+      since ||= Rails.application.config.pod.days_to_fetch.days.ago
+      @file_list = incoming_file_list || AlmaPodFileList.new(file_pattern: file_pattern, since: since)
+      clean_documents
```

Comment:
@sandbergja Only a slight concern that you are setting an instance variable here and above you are setting the instance variable in the AlmaPodFileList constructor.  I think I prefer the pattern in AlmaPodFileList, but I am just noting the inconsistency.  I'm still going to approve and merge this PR...